Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ruby:2.6
```

Comment:
is there any clever way to read the version from `.ruby-version`, so we don't need to maintain in multiple places.

Reply:
Could go in a .env file?

Reply:
we can read it from 
`export RUBY_VERSION=`cat .ruby-version`